### PR TITLE
sql: add the description of small transactions

### PR DESCRIPTION
**Pull request by CaitinChen**
via https://github.com/pingcap/docs/pull/714
@lilin90 @kennytm PTAL

### CODE REVIEW

File: sql/mysql-compatibility.md [hunk 1]
```diff
@@ -109,6 +109,28 @@ TiDB 使用乐观事务模型，在执行 Update、Insert、Delete 等语句时
 * 键值对的总数不超过 300,000
 * 键值对的总大小不超过 100MB
 
+### 小事务
+
+由于 TiDB 中的每个事务都需要跟 PD leader 建立两个 round trip，TiDB 中的小事务相比于 MySQL 中的小事务延迟更高。以如下的 query 为例，除去 `auto_commit`，使用更明确的事务，可优化该 query 的性能。
```
Comment (kennytm) [inline sql/mysql-compatibility.md]:
```suggestion
由于 TiDB 中的每个事务都需要跟 PD leader 进行两次 round trip，TiDB 中的小事务相比于 MySQL 中的小事务延迟更高。以如下的 query 为例，除去 `auto_commit`，使用更明确的事务，可优化该 query 的性能。
```

**Comment by CaitinChen** [comment]
@kennytm Please sign CLA. Thanks.

**Comment by kennytm** [comment]
@CaitinChen Done.

File: sql/mysql-compatibility.md [hunk 2]
```diff
@@ -109,6 +109,28 @@ TiDB 使用乐观事务模型，在执行 Update、Insert、Delete 等语句时
 * 键值对的总数不超过 300,000
 * 键值对的总大小不超过 100MB
 
+### 小事务
+
+由于 TiDB 中的每个事务都需要跟 PD leader 进行两次 round trip，TiDB 中的小事务相比于 MySQL 中的小事务延迟更高。以如下的 query 为例，除去 `auto_commit`，使用更明确的事务，可优化该 query 的性能。
```
Comment (lilin90) [inline sql/mysql-compatibility.md]:
```suggestion
由于 TiDB 中的每个事务都需要跟 PD leader 进行两次 round trip，TiDB 中的小事务相比于 MySQL 中的小事务延迟更高。以如下的 query 为例，用显示事务代替 `auto_commit`，可优化该 query 的性能。
```

File: sql/mysql-compatibility.md [hunk 3]
```diff
@@ -109,6 +109,28 @@ TiDB 使用乐观事务模型，在执行 Update、Insert、Delete 等语句时
...
+
+```sql
+# 带 auto_commit 的原始版本
```
Comment (lilin90) [inline sql/mysql-compatibility.md]:
```suggestion
# 使用 auto_commit 的原始版本
```

**Comment by CaitinChen** [comment]
@lilin90 Thanks. PTAL again

**Pull request merged**